sql/backfill.go

```diff
@@ -188,18 +193,44 @@ func (sc *SchemaChanger) getTableSpan() (roachpb.Span, error) {
 	}, nil
 }
 
+// backoffDurationDuringBackfill is the approximate time that the backfill
+// suspends activity between steps of the backfill. This value was picked
+// through some experimentation:
+// 1. Run block-writer against a 3 node cluster on GCE.
+// 2. Run create/drop index, create/drop column schema changes.
+var backoffDurationDuringBackfill = 75 * time.Millisecond
+
+// Slow down the backfill so that it doesn't rob all the processing power.
+// Do not backoff when backfillHasCompleted.
+func backoffDuringBackfill(backfillHasCompleted bool, testingKnobs *ExecutorTestingKnobs) {
```

Comment:
Passing a `backfillHasCompleted` boolean into this function seems strange. If the backfill is done why haven't the callers just broken out of their loop?